metriq-api/service/platformService.js

```diff
@@ -0,0 +1,207 @@
+// platformService.js
+
+// Data Access Layer
+const ModelService = require('./modelService')
+
+// Database Model
+const db = require('../models/index')
+const Platform = db.platform
```

Comment:
Seems like you typically only use the "capitalize the first letter" convention when doing something like `require('...')`. I'm not sure if this is intentional, but you *might* want to have `const platform ...` instead of `const Platform`?